war/src/main/js/util/page.js

```diff
@@ -41,7 +41,7 @@ function onWinScroll(callback) {
 }
 
 function pageHeaderHeight() {
-    return elementHeight('#page-head');
+    return elementHeight('#page-header');
```

Comment:
I am not confident this is correct. Seems used by `config-scrollspy.js` and starting to scroll on a Jenkins 2.0 style job config form is jarring in the PR build.

Reply:
Good spot! It was missing the addition of the breadcrumb height, fixed now.